modules/javafx.web/src/main/java/com/sun/webkit/Utilities.java

```diff
@@ -119,17 +116,6 @@ private static Object fwkInvokeWithContext(final Method method,
             });
         }
 
-        try {
-            return AccessController.doPrivileged((PrivilegedExceptionAction<Object>)
-                    () -> MethodHelper.invoke(method, instance, args), acc);
-        } catch (PrivilegedActionException ex) {
-            Throwable cause = ex.getCause();
-            if (cause == null)
-                cause = ex;
-            else if (cause instanceof InvocationTargetException
-                && cause.getCause() != null)
-                cause = cause.getCause();
-            throw cause;
-        }
+        return MethodHelper.invoke(method, instance, args);
```

Comment:
I think the whole fwkInvokeWithContext method and associated constants can be safely removed.

Reply:
Very unlikely, but I'll take a look when fixing [JDK-8342993](https://bugs.openjdk.org/browse/JDK-8342993), since this method is on my list to be modified.

Reply:
I don't see this private method being called...

Reply:
https://github.com/openjdk/jfx/blob/master/modules/javafx.web/src/main/native/Source/WebCore/bridge/jni/jsc/JNIUtilityPrivate.cpp#L325-L330

All Java methods in `javafx.web` that are named `fwk*` are called by native WebKit code via JNI upcalls.

Reply:
ouch.  it would be nice to have the method commented or annotated somehow...
thank you for clarification!

Reply:
Indeed it would. Instead we just have to "know" the convention. Not very satisfying, is it!